src/main/java/com/devcomanda/easymeetup/entity/Meetup.java

```diff
@@ -0,0 +1,110 @@
+package com.devcomanda.easymeetup.entity;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import java.time.LocalDateTime;
+import java.util.Objects;
+
+@Entity
+public class Meetup {
+    @Id
+    @GeneratedValue
+    private Long id;
+    private String meetupName;
+    private String meetupAddress;
+    private LocalDateTime meetupDate;
+    private String meetupDescription;
+    private String meetupSpeaker;
+
+    public Meetup(){
+
+    }
+
+    public Meetup(String meetupName, String meetupAddress, LocalDateTime meetupDate,
```

Comment:
I don't find a place of using that constructor, if you don't use it, please remove it